### PR TITLE
Replace UITableView backgound with AlabasterColor #1947

### LINKED ISSUE #1947: Replace UITableView backgound with AlabasterColor

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #1947

### CODE REVIEW

File: AlphaWallet/Settings/ViewControllers/LocalesViewController.swift [hunk 1]
```diff
@@ -16,15 +16,16 @@ class LocalesViewController: UIViewController {
...
 
         roundedBackground.translatesAutoresizingMaskIntoConstraints = false
         view.addSubview(roundedBackground)
 
         tableView.translatesAutoresizingMaskIntoConstraints = false
         tableView.delegate = self
         tableView.separatorStyle = .singleLine
-        tableView.backgroundColor = Colors.appBackground
+        tableView.backgroundColor = GroupedTable.Color.background
+        tableView.tableFooterView = UIView()
```
Comment (hboon) [inline AlphaWallet/Settings/ViewControllers/LocalesViewController.swift:28]: What does the footer do?

Comment (vladyslav-iosdev) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift:28]: @hboon  I added it to make these table view screens look the same like settings screen, it removes separator lines for empty cells

Comment (hboon) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift:28]: Oh very nice! Can you add a new static function to `UIView` and put it somewhere and call it, for future documentation. Something like:

```
static func tableFooterToRemoveEmptyCellSeparators() -> UIView {
  return .init()
}
```

Comment (vladyslav-iosdev) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift]: @hboon Done

Comment (hboon) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift]: The one in `AssetDefinitionsOverridesViewController.swift` too :)

Comment (vladyslav-iosdev) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift]: > AssetDefinitionsOverridesViewController

oooh sorry, i missed it.

Comment (vladyslav-iosdev) [reply AlphaWallet/Settings/ViewControllers/LocalesViewController.swift]: @hboon check this one, should be done.

**Pull request merged**